Makefile

```diff
@@ -7,6 +7,11 @@ help:
 AGGREGATOR_ECDSA_PRIV_KEY=0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6
 CHALLENGER_ECDSA_PRIV_KEY=0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a
 
+ETH_RPC_URL=http://localhost:8545
+ETH_WS_URL=ws://localhost:8545
+
+AGGREGATOR_SERVER_IP_PORT_ADDRESS=localhost:8090
```

Comment:
This is going to get out of hands. Let's move the config variables to a .env file that users can source before running make commands. ALso this way we can have an anvil.env file and a holesky.env file, etc.